pkg/store/event/event.go

```diff
@@ -44,15 +44,106 @@ type eventStore struct {
 	db *dbstore.DB
 }
 
-// TODO: implement core.EventStore interface
-func (e *eventStore) List(context.Context) ([]*core.Event, error) { return nil, nil }
-func (e *eventStore) ListByExperiment(context.Context, string, string) ([]*core.Event, error) {
-	return nil, nil
+// FindPodsByEventID return the list of PodRecords according to the eventID
+func (e *eventStore) FindPodsByEventID(_ context.Context, id uint) ([]*core.PodRecord, error) {
```

Comment:
I think `FindPodsByEventID` is misleading. What about `FindPodRecordsByEventID` ?